include/mbedtls/mbedtls_config.h

```diff
@@ -3325,7 +3327,7 @@
  * armclang <= 6.9
  *
  * \note This was previously known as MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY.
- * That name is deprecated, but may still be used as an alternative form for this
+ * That name is \deprecated, but may still be used as an alternative form for this
```

Comment:
How does `\deprecated` work in Doxygen? If it refers to the thing we're currently documenting, which is `MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_ONLY`, then we shouldn't make it here.

I guess the question is: how does Doxygen know that `\deprecated` refers to `MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY` not `MBEDTLS_SHA256_USE_ARMV8_A_CRYPTO_ONLY`?

Reply:
Good point. Doxygen says: `Starts a paragraph indicating that this documentation block belongs to a deprecated entity.` So I think this is wrong.